### ISSUE TITLE
Batch Delete Expired Entitlements

### ISSUE THREAD

**Issue by timcortesi** [BinghamtonUniversity/IAMBing#16]

**Comment by timcortesi** [comment]
@alikemaltanriverdi Can you please re-verify that this works?  I refactored and simplified this code a couple weeks ago to remove redundancy.

Take a look at Kernel.php

```php
        $schedule->call(function(){
            $identity_ids = IdentityEntitlement::where('type','add')->where('expiration_date',"<",Carbon::now())->get()->pluck('identity_id')->unique();    
            foreach($identity_ids as $identity_id){
                UpdateIdentityJob::dispatch([
                    'identity_id' => $identity_id
                ]);
            }
        })->name('delete_expired_identity_entitlements')->dailyAt(config('app.delete_expired_identity_entitlements'))->timezone('America/New_York')
        ->onOneServer();
```

**Comment by timcortesi** [comment]
Should be fixed by 4f246b8f3b9c35eb40fc56c8cf40fe4c2ee93728

@alikemaltanriverdi please verify

**Comment by alikemaltanriverdi** [comment]
This is working as expected
